modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/StructuredContentResourceTest.java

```diff
@@ -1035,6 +1081,81 @@ private StructuredContentResource _buildStructureContentResource(
 		).build();
 	}
 
+	private JSONArray _createBatchBody(long structureId) throws Exception {
+		StructuredContent structuredContent = new StructuredContent();
```

Comment:
I don't understand why you are creating here a StructuredContent that has been previously created with the _randomStructuredContent method, and I think that random already includes a Content Field, what it would make the issue reproducible already. I think I mentioned in a previous comment that it is possible to use the toString method. Something like:

```java
JSONFactoryUtil.createJSONArray(randomStructuredContent1.toString())
```